Classes/ObjectExplorers/Sections/Shortcuts/FLEXLayerShortcuts.m

```diff
@@ -15,11 +15,11 @@ @implementation FLEXLayerShortcuts
 + (instancetype)forObject:(CALayer *)layer {
     return [self forObject:layer additionalRows:@[
         [FLEXActionShortcut title:@"Preview Image" subtitle:nil
-            viewer:^UIViewController *(id layer) {
+            viewer:^UIViewController *(CALayer *layer) {
```

Comment:
Does the compiler not warn or complain about this? If not, nice!

Reply:
It doesn't warn on compiling :)